metadata-ingestion/src/datahub/ingestion/source/sql/mysql.py

```diff
@@ -83,3 +84,15 @@ def get_platform(self):
     def create(cls, config_dict, ctx):
         config = MySQLConfig.parse_obj(config_dict)
         return cls(config, ctx)
+
+    def add_profile_metadata(self, inspector: Inspector) -> None:
+        with inspector.engine.connect() as conn:
+            for row in conn.execute(
+                "SELECT table_schema, table_name, data_length from information_schema.tables"
+            ):
+                table_schema = row[0]
```

Comment:
Can we use column names here instead of positions?

Reply:
Is that even supported?

Reply:
It should, I tried and it worked either with `row['table_schema']` or with `row.table_schema`